packages/lesswrong/components/tagging/FooterTag.tsx

```diff
@@ -144,9 +156,13 @@ const FooterTag = ({
   neverCoreStyling?: boolean,
   hideRelatedTags?: boolean,
   className?: string,
-  noBackground?: boolean
-  classes: ClassesType<typeof styles>,
-}) => {
+  noBackground?: boolean,
+} & (
+    { hoverable: false, tag: TagBasicInfo }
+  | { hoverable: true, tag: TagPreviewFragment }
+  | { hoverable: "ifDescriptionPresent", tag: TagBasicInfo|TagPreviewFragment }
```

Comment:
Minor stylistic thing: I think this would be better as "always", "never", or "ifDescriptionPresent", rather than mixing string and boolean types. Partly to nudge the developer that non-boolean options exist (and partly because the mixed type gives me bad vibes)